routes/cube_routes.js

```diff
@@ -915,9 +891,11 @@ router.get('/samplepackimage/:id/:seed', function(req, res) {
       }).then(function(image) {
         res.writeHead(200, {
           'Content-Type': 'image/png',
-          'Content-Length': image.length
+          'Content-Encoding': 'gzip'
+        });
+        zlib.gzip(Buffer.from(image.replace(/^data:image\/png;base64,/, ''), 'base64'), function(_, result) {
+          res.end(result);
```

Comment:
Don't we want the webserver to be doing the compression? Also isn't it not going to accomplish much to compress an image that is already in a compressed format?

Reply:
What format is that? base64? Image previews require gzip or another specific compression method

Reply:
Refactored this, you're right that it's already compressed, as gzip. We just need to add the encoding into the header, and add og:width and og:height to make this work.